<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>667</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Easy Morse</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/gfx/Morse_128x128.png</normaloff>:/gfx/Morse_128x128.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="m_leMessage"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_pbSend">
        <property name="maximumSize">
         <size>
          <width>16777212</width>
          <height>16777213</height>
         </size>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="m_LogZone">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>72</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionafOptions"/>
   <addaction name="afAbout"/>
  </widget>
  <widget class="QDockWidget" name="dockGlossary">
   <property name="windowTitle">
    <string>Glossary</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="vlGlossaire">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QTableWidget" name="m_TableGlossaire">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row>
        <property name="text">
         <string notr="true">A</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">B</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">C</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">D</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">E</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">F</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">G</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">H</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">I</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">J</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">K</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">L</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">M</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">N</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">O</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">P</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">Q</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">R</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">S</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">T</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">U</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">V</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">W</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">X</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">Y</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">Z</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">0</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string notr="true">9</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Morse</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string notr="true">a</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignVCenter</set>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string notr="true">b</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string notr="true">c</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string notr="true">d</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string notr="true">e</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string notr="true">f</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string notr="true">g</string>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string notr="true">h</string>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string notr="true">i</string>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string notr="true">j</string>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string notr="true">k</string>
        </property>
       </item>
       <item row="11" column="0">
        <property name="text">
         <string notr="true">l</string>
        </property>
       </item>
       <item row="12" column="0">
        <property name="text">
         <string notr="true">m</string>
        </property>
       </item>
       <item row="13" column="0">
        <property name="text">
         <string notr="true">n</string>
        </property>
       </item>
       <item row="14" column="0">
        <property name="text">
         <string notr="true">o</string>
        </property>
       </item>
       <item row="15" column="0">
        <property name="text">
         <string notr="true">p</string>
        </property>
       </item>
       <item row="16" column="0">
        <property name="text">
         <string notr="true">q</string>
        </property>
       </item>
       <item row="17" column="0">
        <property name="text">
         <string notr="true">r</string>
        </property>
       </item>
       <item row="18" column="0">
        <property name="text">
         <string notr="true">s</string>
        </property>
       </item>
       <item row="19" column="0">
        <property name="text">
         <string notr="true">t</string>
        </property>
       </item>
       <item row="20" column="0">
        <property name="text">
         <string notr="true">u</string>
        </property>
       </item>
       <item row="21" column="0">
        <property name="text">
         <string notr="true">v</string>
        </property>
       </item>
       <item row="22" column="0">
        <property name="text">
         <string notr="true">w</string>
        </property>
       </item>
       <item row="23" column="0">
        <property name="text">
         <string notr="true">x</string>
        </property>
       </item>
       <item row="24" column="0">
        <property name="text">
         <string notr="true">y</string>
        </property>
       </item>
       <item row="25" column="0">
        <property name="text">
         <string notr="true">z</string>
        </property>
       </item>
       <item row="26" column="0">
        <property name="text">
         <string notr="true">0</string>
        </property>
       </item>
       <item row="27" column="0">
        <property name="text">
         <string notr="true">1</string>
        </property>
       </item>
       <item row="28" column="0">
        <property name="text">
         <string notr="true">2</string>
        </property>
       </item>
       <item row="29" column="0">
        <property name="text">
         <string notr="true">3</string>
        </property>
       </item>
       <item row="30" column="0">
        <property name="text">
         <string notr="true">4</string>
        </property>
       </item>
       <item row="31" column="0">
        <property name="text">
         <string notr="true">5</string>
        </property>
       </item>
       <item row="32" column="0">
        <property name="text">
         <string notr="true">6</string>
        </property>
       </item>
       <item row="33" column="0">
        <property name="text">
         <string notr="true">7</string>
        </property>
       </item>
       <item row="34" column="0">
        <property name="text">
         <string notr="true">8</string>
        </property>
       </item>
       <item row="35" column="0">
        <property name="text">
         <string notr="true">9</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockOptions">
   <property name="minimumSize">
    <size>
     <width>166</width>
     <height>543</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="vlOptions">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Output Sound Card</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_deviceBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Serial Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_serialBox"/>
     </item>
     <item>
      <widget class="QLabel" name="m_labelFreq">
       <property name="text">
        <string>Sound Freq=%1 Hz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_frequencySlider">
       <property name="minimum">
        <number>200</number>
       </property>
       <property name="maximum">
        <number>1200</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>200</number>
       </property>
       <property name="value">
        <number>600</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickInterval">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelSoundLevel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Sound Level = %1 %</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_volumeSlider">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelWordSpeed">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Word Speed = %1 WPM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_WordSpeed">
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="sliderPosition">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelNoiseMixing">
       <property name="text">
        <string>Noise Mixing = %1 %</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_NoiseCorr">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_LowPassFilter">
       <property name="text">
        <string>Low Pass Filter = %1 %</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_NoiseFilterSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_UseFarnsWorth">
       <property name="text">
        <string>Use Farnsworth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_labelCharSpeed">
       <property name="text">
        <string>Char Speed = %1 WPM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_CharSpeed">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockDecode">
   <property name="windowTitle">
    <string>Morse Decode</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionafOptions">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/gfx/Outils.png</normaloff>:/gfx/Outils.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="toolTip">
    <string>Options</string>
   </property>
  </action>
  <action name="afAbout">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/gfx/help.png</normaloff>:/gfx/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
